api/lib/opentelemetry/trace/tracer.rb

```diff
@@ -40,6 +46,17 @@ def start_root_span(name, attributes: nil, links: nil, start_timestamp: nil, kin
         Span.new
       end
 
+      # Used when a caller wants to manage the activation/deactivation and lifecycle of
+      # the Span and its parent manually.
+      #
+      # Parent context can be either passed explicitly, or inferred from currently activated span.
+      #
+      # @param [optional Span] with_parent Explicitly managed parent Span, overrides
+      #   +with_parent_context+.
+      # @param [optional SpanContext] with_parent_context Explicitly managed. Overridden by
+      #   +with_parent+.
+      #
+      # @return [Span]
       def start_span(name, with_parent: nil, with_parent_context: nil, attributes: nil, links: nil, start_timestamp: nil, kind: nil, sampling_hint: nil)
```

Comment:
Should we document all the other params here and for `in_span` above in this PR or would you rather address that separately?

Reply:
I don't think I have enough context yet to fully understand the other params, so "separately" would be better for me at this point.